### PR TITLE
Updates to work with Centos7 cloud image

### PR DESCRIPTION
**Pull request by tdeckers**
Ensure NetworkManager package is installed.
Ensure dnsmasq.d directory exists.  
Set proper permission on 10-consul - it won't execute otherwise (dnsmasq fails to start if 10-consul isn't executable)
Ensure to restart NetworkManager after reconfigurations

### CODE REVIEW

File: roles/dnsmasq/tasks/main.yml [hunk 1]
```diff
@@ -15,6 +24,7 @@
   template: 
     src: 10-consul 
     dest: /etc/NetworkManager/dnsmasq.d/10-consul
+    mode: 777
```
Comment (stevendborrelli) [inline roles/dnsmasq/tasks/main.yml]: Can we have tighter permissions on this file? 755 is fine.

**Comment by keithchambers** [comment]
Testing this now ...

**Comment by keithchambers** [comment]
LGTM ... merging!  Thank you!

**Comment by keithchambers** [comment]
LGTM ... merging! Thank you!

**Pull request merged**